modules/ob_common/manifests/params.pp

```diff
@@ -45,64 +45,72 @@
   $server_script_path = "${product_dir}/${pack}/bin/wso2server.sh"
   $pid_file_path      = "${product_dir}/${pack}/wso2carbon.pid"
 
-  # Hostname changes in apimanager.xml, openbanking.xml, carbon.xml, identity.xml, application-authentication.xml, conf.json params and velocity_template.xml params
+  # Hostname changes in deployment.toml, conf.json params and velocity_template.xml params
   $iam_hostname       = 'localhost'
   $analytics_hostname = 'localhost'
   $apim_hostname      = 'localhost'
   $bps_hostname       = 'localhost'
 
-  # jaggeryapps/admin/site/conf/site.json,openbaking.xml params
+  # jaggeryapps/admin/site/conf/site.json,deployment.toml params
   $spec = 'UK' #UK,Berlin or STET 
   
   # DBMS related variables
 
   # master-datasources.xml params
-  $db_apimgt_url              = 'jdbc:h2:repository/database/WSO2AM_DB;DB_CLOSE_ON_EXIT=FALSE'
-  $db_apimgt_username         = 'wso2carbon'
-  $db_apimgt_password         = 'wso2carbon'
-  $db_apimgt_validation_query = 'SELECT 1'
-  $db_apimgt_driver           = 'org.h2.Driver'
-
-  $db_mb_store_url              = 'jdbc:h2:repository/database/WSO2MB_DB;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=60000'
-  $db_mb_store_username         = 'wso2carbon'
-  $db_mb_store_password         = 'wso2carbon'
-  $db_mb_store_validation_query = 'SELECT 1'
-  $db_mb_store_driver           = 'org.h2.Driver'
-
-  $db_apimgt_stat_url               = 'jdbc:h2:repository/database/WSO2_APIMGT_STAT_DB;DB_CLOSE_ON_EXIT=FALSE'
-  $db_apimgt_stat_username          = 'wso2carbon'
-  $db_apimgt_stat_password          = 'wso2carbon'
-  $db_apimgt_stat_validation_query  = 'SELECT 1'
-  $db_apimgt_stat_driver            = 'org.h2.Driver'
+  $db_apimgt_url                     = 'jdbc:h2:repository/database/WSO2AM_DB;DB_CLOSE_ON_EXIT=FALSE'
+  $db_apimgt_username                = 'wso2carbon'
+  $db_apimgt_password                = 'wso2carbon'
+  $db_apimgt_validation_query        = 'SELECT 1'
+  # Use below for oracle
+  # $db_apimgt_validation_query      = 'SELECT 1 FROM DUAL'
```

Comment:
Shall we add this to the Readme under a new topic - 'customizations'? The validation query is not the only thing that changes when you change the DB to Oracle.

Reply:
done! Mentioned the database type specific config changes in a separate section of the readMe file